docs/specification/current.md

```diff
@@ -733,8 +720,11 @@ POST {baseUrl}/cds-services/{serviceId}/feedback
          "card":"9368d37b-283f-44a0-93ea-547cebab93ed",
          "outcome":"overridden",
          "overrideReason": { 
-	 	"code":"reason-code-provided-by-service",
-     		"system":"http://example.org/cds-services/fhir/CodeSystem/override-reasons",
+	 	"reason": {
+	 		"code":"reason-code-provided-by-service",
+     			"system":"reason-system-provided-by-service",
+     			"display":"reason-display-provided-by-service",
```

Comment:
nit: there isn't any value I can think of to sending the display to the feedback endpoint.  It's certainly benign and shows the full Coding structure here, but it also touches on the difference between making an API call and sending a FHIR resource.  I think the example should reflect that a CDS service just needs to know the machine-readable data.